webui/react/src/pages/DesignKit.tsx

```diff
@@ -2303,6 +2316,18 @@ const IconsSection: React.FC = () => {
             <Icon key={name} name={name} showTooltip title={name} />
           ))}
         </Space>
+        <br />
+        <p>SVG Icons usage.</p>
+        <Space wrap>
```

Comment:
You don't need to separate these out from the other icons. The fact that they're SVG should be an implementation detail.